package/contents/ui/main.qml

```diff
@@ -316,39 +348,38 @@ Item {
 				// See: http://qt-project.org/doc/qt-5/qml-qtquick-wheelevent.html#angleDelta-prop
 				while (wheelDelta >= 120) {
 					wheelDelta -= 120
-					widget.performMouseWheelUp()
+					root.performMouseWheelUp()
 				}
 				while (wheelDelta <= -120) {
 					wheelDelta += 120
-					widget.performMouseWheelDown()
+					root.performMouseWheelDown()
```

Comment:
It's a QtQuick standard to use `root` for the top most QML item, however in my experience `root` is reused a few times in the project. Also, in the grand scheme of things, this widget *isn't* the root QML object since there's the panel/desktop QML object that wraps it.

I've run into this when I imported the calendar's `MonthView` and found that `root` wasn't pointing to the plasmoid root item, but the calendar itself...

* https://invent.kde.org/plasma/plasma-workspace/-/blob/master/components/calendar/qml/MonthView.qml#L16

I'll be renaming this back to `widget` after merging.